cmd/influxd/run.go

```diff
@@ -93,6 +94,16 @@ func execRun(args []string) {
 		}
 		log.Printf("DataNode#%d running on %s", s.ID(), config.ApiHTTPListenAddr())
 
+		// Spin up the collectd server
+		if config.Collectd.Enabled {
+			c := config.Collectd
+			s := collectd.NewServer(s, c.TypesDB)
```

Comment:
I would use a different variable name here, instead of `s`. It looks super-weird, and is not obvious that we are talking about two different `s`s here.